Application/EdFi.Ods.WebApi.NetCore/Program.cs

```diff
@@ -34,7 +34,16 @@ public static async Task Main(string[] args)
                             .AddEnvironmentVariables();
                     })
                 .UseServiceProviderFactory(new AutofacServiceProviderFactory())
-                .ConfigureWebHostDefaults(webBuilder => { webBuilder.UseStartup<Startup>(); }).Build();
+                .ConfigureWebHostDefaults(
```

Comment:
we should add this change to the startup for the SwaggerUI project too please

Reply:
okay Jim I will add it